explainer.md

```diff
@@ -122,7 +120,7 @@ document.getElementById('changeConfigBtn').onclick() = function() {
     return;
   }
 
-  if (connection.url != '_self') {
+  if (connection.type != 'mirroring') {
```

Comment:
Looking at the rest of the code below:

- If the connection.type is mirroring, what does that imply about the state of `latencyHint` and `audioPlayback`? If they can never be undefined, then the logic below can be simplified.
- Do you need to change `connection.source.latencyHint` to `connection.latencyHint` below?

Reply:
- Right, done.
- Done.